src/Contracts/DataModels.ts

```diff
@@ -199,11 +200,19 @@ export interface IndexingPolicy {
   compositeIndexes?: any[];
   spatialIndexes?: any[];
   vectorIndexes?: VectorIndex[];
+  fullTextIndexes?: FullTextIndex[];
 }
 
 export interface VectorIndex {
   path: string;
   type: "flat" | "diskANN" | "quantizedFlat";
+  diskANNShardKey?: string;
```

Comment:
Shard key applies to both diskANN and quantizedFlat.

Also, we aren't shipping this next week because we don't have it deployed.

Reply:
In the specs I have the field is still called "diskANNShardKey". I will leave this until the feature is released and edit if necessary. This field isn't visible anywhere in the ux yet.